test/images/image-locality-test/pod.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: hostexec
+  labels:
+    app: hostexec
+spec:
+  containers:
+  - name: hostexec
+    image: gcr.io/kubernetes-e2e-test-images/image-locality-test-amd64:1.0
```

Comment:
now part of `make all-push` target we are pushing fat manifest, so you don't have to mention `-amd64` in the end!

Reply:
Sure. Thanks!